packages/react-core/src/components/Nav/NavExpandable.tsx

```diff
@@ -126,14 +126,14 @@ class NavExpandable extends React.Component<NavExpandableProps, NavExpandableSta
             <PageSidebarContext.Consumer>
               {({ isSidebarOpen }) => (
                 <button
-                  className={styles.navLink}
+                  className={css(styles.navLink)}
                   id={srText ? null : this.id}
                   onClick={(event) => this.onExpand(event, context.onToggle)}
                   aria-expanded={expandedState}
                   tabIndex={isSidebarOpen ? null : -1}
                   {...buttonProps}
                 >
-                  {title}
+                  {typeof title !== 'string' ? <span className={css(`${styles.nav}__link-text`)}>{title}</span> : title}
```

Comment:
lgtm, only other thing is that we're having this wrapper be an opt-in for the NavItem, but providing logic here to determine if the wrapper is used or not. It seems like we'd want it to be opt in in both places?

Reply:
So it is not opt in here because we did not have the ability to pass a react node before.  Menaing. that it is a new feature.  We will not be breaking anyone.

Reply:
Can we open an issue to make this the default in the next breaking change, if there isn't one already?

Reply:
@mcoker breaking change issue opened https://github.com/patternfly/patternfly-react/issues/9755